cumulus_library/template_sql/codeable_concept_denormalize.sql.jinja

```diff
@@ -54,6 +54,7 @@ CREATE TABLE {{ target_table }} AS (
             ROW_NUMBER()
                 OVER (
                     PARTITION BY id
+                    ORDER BY priority ASC
```

Comment:
This was a bug found during testing -- without this order-by, you can get inconsistent results run-to-run